core/domain/suggestion_services_test.py

```diff
@@ -2273,48 +2433,127 @@ def test_update_question_stats(self):
             )
         )
-        question_contribution_stats_model = (
-            suggestion_models.QuestionContributionStatsModel.get(
-                self.author_id, topic_id
-            )
-        )
 
         self.assertEqual(
-            question_contribution_stats_model.submitted_questions_count,
-            1
+            question_review_stats_model.accepted_questions_count,  # type: ignore[union-attr]
+            2
         )
         self.assertEqual(
-            question_contribution_stats_model.accepted_questions_count,
-            1
+            (
+                question_review_stats_model  # type: ignore[union-attr]
+                .reviewed_questions_count
+            ),
+            2
         )
+
+    def test_update_question_review_stats(self) -> None:
+        skill_id_1 = skill_services.get_new_skill_id()
+        skill_id_2 = skill_services.get_new_skill_id()
+        self.save_new_skill(  # type: ignore[no-untyped-call]
+            skill_id_1, self.author_id, description='description')
+        topic_id = topic_fetchers.get_new_topic_id()
+        self.save_new_topic(  # type: ignore[no-untyped-call]
+            topic_id, 'topic_admin', name='Topic1',
+            abbreviated_name='topic-three', url_fragment='topic-three',
+            description='Description',
+            canonical_story_ids=[],
+            additional_story_ids=[],
+            uncategorized_skill_ids=[skill_id_1, skill_id_2],
+            subtopics=[], next_subtopic_id=1)
+        suggestion_change_1 = {
+            'cmd': (
+                question_domain
+                .CMD_CREATE_NEW_FULLY_SPECIFIED_QUESTION),
+            'question_dict': {
+                'question_state_data': self._create_valid_question_data(
+                    'default_state').to_dict(),
+                'language_code': 'en',
+                'question_state_data_schema_version': (
+                    feconf.CURRENT_STATE_SCHEMA_VERSION),
+                'linked_skill_ids': ['skill_1'],
+                'inapplicable_skill_misconception_ids': ['skillid12345-1']
+            },
+            'skill_id': skill_id_1,
+            'skill_difficulty': 0.3
+        }
+        suggestion_change_2 = {
+            'cmd': (
+                question_domain
+                .CMD_CREATE_NEW_FULLY_SPECIFIED_QUESTION),
+            'question_dict': {
+                'question_state_data': self._create_valid_question_data(
+                    'default_state').to_dict(),
+                'language_code': 'en',
+                'question_state_data_schema_version': (
+                    feconf.CURRENT_STATE_SCHEMA_VERSION),
+                'linked_skill_ids': ['skill_2'],
+                'inapplicable_skill_misconception_ids': ['skillid12345-1']
+            },
+            'skill_id': skill_id_2,
+            'skill_difficulty': 0.3
+        }
+        initial_suggestion = suggestion_services.create_suggestion(  # type: ignore[call-overload]
```

Comment:
Any type ignores (except for no-untyped-call) need a clear explanatory comment. Please talk to @sahiljoster32 for advice on this (or, ideally, update the code so that the ignores are not required).

Reply:
I removed almost every ignored checks/explained in comments but this one and another remains and I will contact @sahiljoster32 for some advice.

Reply:
I have replied to your email. But if you still have any doubts I'll be happy to help.

Reply:
Yes it was resolved. Thanks